tests/modelbench_tests/test_run.py

```diff
@@ -273,7 +273,7 @@ def test_benchmark_bad_sut_errors_out(self, runner, tmp_path):
         benchmark_options.extend(["--locale", "en_us"])
         benchmark_options.extend(["--prompt-set", "practice"])
 
-        with pytest.raises(SUTNotFoundException):
+        with pytest.raises(ValueError, match="No registration for bogus"):
```

Comment:
What's the reason to test for a more generic exception? SUTNotFoundException and related exception classes are more expressive IMO.